modules/oe_theme_content_publication/oe_theme_content_publication.module

```diff
@@ -7,9 +7,53 @@
 
 declare(strict_types = 1);
 
+use Drupal\Core\Cache\CacheableMetadata;
+
 /**
  * Implements template_preprocess_node() for the Publication node type.
  */
 function oe_theme_content_publication_preprocess_node__oe_publication__full(&$variables) {
   $variables['#attached']['library'][] = 'oe_theme_content_publication/oe_publication';
 }
+
+/**
+ * Implements template_preprocess_field().
+ *
+ * If any of the referenced contacts is an organisation we need to first check
+ * if that organisation has any contact details and prevent it from rendering
+ * if it does not.
+ */
+function oe_theme_content_publication_preprocess_field__node__oe_publication_contacts(&$variables) {
+  if (empty($variables['items'])) {
+    return;
+  }
+  foreach ($variables['items'] as &$item) {
+    /** @var \Drupal\Core\Entity\ContentEntityInterface $contact */
+    $contact = $item['content']['#oe_contact'];
+    $cacheable_metadata  = CacheableMetadata::createFromRenderArray($item['content']);
+    // If the contact is not an organisation we don't care.
+    if ($contact->bundle() !== 'oe_organisation_reference') {
+      continue;
+    }
+    $organisation = $contact->oe_node_reference->entity;
+    if (!$organisation) {
```

Comment:
Let's use `!$organisation instanceof ...` here as we use it as an object down below. This will also help with autosuggesting in phpstorm.